docker-compose.yml

```
@@ -2,8 +2,9 @@ services:
   pyrsia:
     image: pyrsia/node
     build:
+      context: .
       target: node
-    command: ["-p", "7878"]
+    command: ["-h", "0.0.0.0", "-p", "7878"]
```

Shouldn't the host parameter be capital h, or is it case insensitive? 
```suggestion
    command: ["-H" , "0.0.0.0", "-p", "7878"]
```